GitForWindowsHelper/component-updates.js

```diff
@@ -12,7 +12,8 @@ const guessComponentUpdateDetails = (title, body) => {
 
     version = version
         .replace(/^(GCM |openssl-|OpenSSL_|v|V_|GnuTLS |tig-|Heimdal |cygwin-|PCRE2-|Bash-)/, '')
-        .replace(/_|\s+patch\s+/, '.')
+        .replace(/\s+patch\s+/, '.')
+        .replace(/_/g, '.')
```

Comment:
We'll need to modify https://github.com/git-for-windows/git-for-windows-automation/blob/f2af655f51f54360d387f3e42dd6f7327ed640c4/update-scripts/version/openssh#L7 accordingly.

Reply:
Good point!!!

Reply:
@rimrul I addressed this in https://github.com/git-for-windows/git-for-windows-automation/pull/27/commits/cb2053fe974cbce4ea00f640ab46d10376e7e0cb